CVLByExample/Curve/README.md

```diff
@@ -0,0 +1,35 @@
+
+#Curve 
+
+This directory contains the `Curve` contract which has the read only reentrancy weakness. 
+
+##Contracts
+The original contract is `Curve`` whose weakness is found by the builtin rule viewReentrancy. In addition we have the
```

Comment:
I'd clarify what was munged in MungedCurve, or that this weakness is detectable *only * by a builtin rule (Expressiveness needed to catch accurately doesn't exist yet in CVL, so it's possible to do some non code affecting munging to get the desirable effect with normal hooks).